project.clj

```diff
@@ -1,7 +1,7 @@
 (def ks-version "1.2.0")
 (def tk-version "1.2.0")
 
-(defproject puppetlabs/trapperkeeper-metrics "0.1.3-SNAPSHOT"
+(defproject puppetlabs/trapperkeeper-metrics "0.2.0-SNAPSHOT"
```

Comment:
I feel like at one point @KevinCorcoran used that and it did something unexpected? But I also don't remember that clearly.